src/StrictJsonTests/ModuleInit.cs

```diff
@@ -3,16 +3,13 @@ public static class ModuleInit
     #region UseStrictJson
 
     [ModuleInitializer]
-    public static void Init()
-    {
+    public static void Init() =>
         VerifierSettings.UseStrictJson();
-    }
+
     #endregion
-    
+
     [ModuleInitializer]
-    public static void InitDerivePathInfo()
-    {
+    public static void InitDerivePathInfo() =>
```

Comment:
Ah, nice. I didn't realize multiple module initializers would work.
I'm assuming the order doesn't matter, right?

Reply:
yeah in this case order doesnt matter